locations/spiders/mattress_firm_us.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from locations.pipelines.phone_clean_up import PhoneCleanUpPipeline
+from locations.storefinders.where2getit import Where2GetItSpider
+
+
+class MattressFirmUSSpider(Where2GetItSpider):
+    name = "mattress_firm_us"
+    api_endpoint = "https://api.slippymap.com/mattressfirmsites/rest/getlist"
+    api_key = "88FD3C6E-2B22-11EE-86CD-EF1E9DC6E625"
+
+    def parse_item(self, item, location, **kwargs):
+        if location["mattress_firm_clearance_center"] == "yes":
+            item["brand"] = "Mattress Firm Clearance"
+            item["brand_wikidata"] = "Q6791878"
+            item["nsi_id"] = "mattressfirmclearance-032196"
+        elif location["sleep_expert_store"] == "yes":
+            item["brand"] = "Sleep Experts"
+            item["brand_wikidata"] = "Q7539688"
+            item["nsi_id"] = "sleepexperts-032196"
+        else:
+            item["brand"] = "Mattress Firm"
+            item["brand_wikidata"] = "Q6791878"
+            item["nsi_id"] = "mattressfirm-032196"
+        item["extras"]["shop"] = "bed"
+        attributes = {kv["id"]: kv["value"] for kv in json.loads(location["attributes"])}
+        item["extras"]["delivery"] = attributes.get("has_delivery")
```

Comment:
Can you use `apply_yes_no()` for this? See https://github.com/alltheplaces/alltheplaces/blob/e16650b807c87325a759d907221d66706b3a1185/locations/spiders/bp.py#L27 for example.